### DISCUSSION TITLE
Button to scroll (smoothly) to a specific slace on my page

### DISCUSSION THREAD

**Discussion by juandaherrera** [reflex-dev/reflex#2476] [Q&A] [answered]
Hi! I'm trying to figure out how to make a button that, when pressed, scrolls down to a specific component at the bottom of my page. Do you know how I could do this in Reflex?

What I'm currently doing is using an href in my button like this:
```python
rx.link(
    rx.button(
        # Button code ...
    ),
    href="#component_id_where_i_want_to_scroll",
)
```

It currently works, but not the way I would like. I'd prefer the scroll to happen smoothly, not instantly. It's important to note that I added the following to the general styles:
```python
BASE_STYLE = {
    #Other style configs ...
    "scroll_behavior": "smooth",
}
```

App:
```python
app = rx.App(style=BASE_STYLE)
```

**Comment by picklelo** [comment]
Played around with this a bit, but wasn't able to get it to work. From what I've seen online setting the `scroll_behavior` prop should work - perhaps there's something special we need to do for NextJS apps

**Comment by juandaherrera** [comment]
Should we open an issue about this? Or is it really a problem with some dependency? I've been looking for different ways to achieve it and haven't succeeded. I think this is a basic and straightforward feature that shouldn't add complexity for the library users when trying to implementing it.

**Comment by juandaherrera** [comment, marked as answer]
I managed to solve it. What I did was put the following in a CSS file:
```css
* {
    scroll-behavior: smooth !important;
}
```
and then I added it to a list of stylesheets that goes directly in the app's definition:
```python
STYLESHEETS = [
    "/css/loop.css",
    #Other stylesheets...
]

app = rx.App(stylesheets=STYLESHEETS)
```
